salt/profiles/postgresql.sls

```diff
@@ -0,0 +1,39 @@
+profiles-db-user:
+    postgres_user.present:
+        - name: {{ pillar.profiles.db.username }}
+        - encrypted: True
+        - password: {{ pillar.profiles.db.password }}
+        - refresh_password: True
+        - createdb: True
+        {% if salt['elife.cfg']('cfn.outputs.RDSHost') %}
+        # remote psql
+        - db_user: {{ salt['elife.cfg']('project.rds_username') }}
+        - db_password: {{ salt['elife.cfg']('project.rds_password') }}
+        - db_host: {{ salt['elife.cfg']('cfn.outputs.RDSHost') }}
+        - db_port: {{ salt['elife.cfg']('cfn.outputs.RDSPort') }}
+        {% else %}
+        # local psql
+        - db_user: {{ pillar.elife.db_root.username }}
+        - db_password: {{ pillar.elife.db_root.password }}
+        {% endif %}
+        - require:
+            - postgresql-ready
+
+profiles-db:
+    postgres_database.present:
+        - owner: {{ pillar.profiles.db.username }}
```

Comment:
ownership is complex in Postgres because the `project.rds_username` is not a real root user, so it won't be able to delete this database (to do a restore for example) as it is owned by someone else. Solutions to do that are not ready so for now it's ok to use this owner